modules/orientdb/src/main/java/org/testcontainers/containers/OrientDBContainer.java

```diff
@@ -0,0 +1,181 @@
+package org.testcontainers.containers;
+
+import com.github.dockerjava.api.command.InspectContainerResponse;
+import com.orientechnologies.orient.core.db.ODatabaseSession;
+import com.orientechnologies.orient.core.db.ODatabaseType;
+import com.orientechnologies.orient.core.db.OrientDB;
+import com.orientechnologies.orient.core.db.OrientDBConfig;
+import lombok.NonNull;
+import org.apache.commons.io.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.wait.strategy.HttpWaitStrategy;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.containers.wait.strategy.WaitAllStrategy;
+import org.testcontainers.containers.wait.strategy.WaitStrategy;
+
+import java.io.IOException;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.time.Duration;
+import java.util.Optional;
+
+import static java.net.HttpURLConnection.HTTP_OK;
+
+/**
+ * @author robfrank
+ */
+public class OrientDBContainer extends GenericContainer<OrientDBContainer> {
+
+    private static final String DEFAULT_IMAGE_NAME = "orientdb";
+
+    private static final String DEFAULT_TAG = "3.0.24-tp3";
+
+    private static final String DEFAULT_USERNAME = "admin";
+
+    private static final String DEFAULT_PASSWORD = "admin";
+
+    private static final String DEFAULT_DATABASE_NAME = "testcontainers";
+
+    private static final int DEFAULT_BINARY_PORT = 2424;
+
+    private static final int DEFAULT_HTTP_PORT = 2480;
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(OrientDBContainer.class);
+
+    private static final String DOCKER_IMAGE_NAME = DEFAULT_IMAGE_NAME + ":" + DEFAULT_TAG;
+
+    private String databaseName;
+
+    private String username;
+
+    private String password;
+
+    private OrientDB orientDB;
+
+    private ODatabaseSession session;
+
+    private Optional<String> scriptPath = Optional.empty();
```

Comment:
Just a tiny style point, but I'd be inclined to reduce the spacing and reorder some of these fields (to group similar fields together, and to reduce visible space taken).

Reply:
cleaned